packages/lib/jobs/definitions/internal/seal-document.handler.ts

```diff
@@ -493,24 +485,13 @@ export const getCertificateAndAuditLogData = async ({
     : null;
 
   const getAuditLogDataPromise = settings.includeAuditLog
-    ? getAuditLogsPdf({
-        documentId: legacyDocumentId,
-        language: documentMeta.language,
-      }).catch((e) => {
+    ? getAuditLogPdfByPlaywright(payload).catch((e) => {
```

Comment:
for #2251, we can make it switchable like this:
```
? match(env('NEXT_PRIVATE_PDF_GENERATOR'))
      .with('playwright', async () => await getAuditLogPdfByPlaywright(payload))
      .otherwise(async () => await generateAuditLogPdf(payload)).catch((e) => {
```